src/wasm/source-map.cpp

```diff
@@ -135,6 +139,9 @@ SourceMapReader::readDebugLocationAt(size_t currLocation) {
       col += readBase64VLQ();
 
       next = peek();
+      if (next == ';') {
+        throw MapParseException("Unexpected mapping for 2nd generated line");
```

Comment:
What does "2nd" mean here?

Reply:
A generated JS file can have multiple lines. Mappings for each line are separated by a `;` character. But wasm files are treated as if they have only one line, and the column number is treated as the binary offset. I'll add a comment.